src/goto-checker/symex_coverage.cpp

```diff
@@ -52,7 +52,7 @@ class goto_program_coverage_recordt : public coverage_recordt
 public:
   goto_program_coverage_recordt(
     const namespacet &ns,
-    goto_functionst::function_mapt::const_iterator gf_it,
+    const goto_functionst::function_mapt::value_type &gf_entry,
```

Comment:
Feels a little cumbersome.  Isn't this a `goto_functiont`?

Reply:
These also require the name of the `goto_functiont`. Would could be done is passing that information as two arguments? That would probably make for are clearer interface. Let me fix this.